portfolio/src/main/java/com/google/sps/data/CalendarEvent.java

```diff
@@ -0,0 +1,34 @@
+package com.google.sps.data;
+
+/** Models a calendar event that the user creates. */
+public class CalendarEvent {
+  private final String name;
+  private final String startTime;
```

Comment:
Let's indicate the time unit. Ex: startTimeMillis. Also, the type is string. I don't think the type should be String. At least it should be long to store the timestamp millis.

Also, consider using the Instant Java class:
https://docs.oracle.com/javase/8/docs/api/java/time/Instant.html

It might be a good time to use the correct data types, so you don't have to refactor later.

Reply:
In the calendar servlet, I was hoping to use the fromJson method from the Gson library to unpack a JSON String directly into an instance of the CalendarEvent class. It seems easier for this method to work if the time parameters are of type String. The workaround I have right now is, the start and end times parameters are of type String, but the getter methods will use Instant.parse and return an instance of Instant class.